source/management_console/public/js/libapi.js

```diff
@@ -27,96 +27,13 @@
 // This file is borrowed from lynckia/licode with some modifications.
 
 'use strict';
-var encodeBase64 = (function () {
```

Comment:
Since you don't use crypto-js here, you can remove it in related install script.